recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,139 @@
+from conans import ConanFile, CMake, tools
+import os
+import json
+
+
+class VtkConan(ConanFile):
+    name = "vtk"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://vtk.org/"
+    description = "VTK is an open-source software system for" \
+        "image processing, 3D graphics, volume rendering and visualization."
+    topics = ("conan", "vtk", "geometry", "algorithms")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "build_testing": ["ON", "OFF", "WANT"],
+               "vtk_smp_implementation": ["Sequential", "OpenMP", "TBB", "STDThread"],
```

Comment:
Option values should be lower case https://github.com/conan-io/conan-center-index/blob/41d4fbb8c218f9d938ccb8b9d888bff52002e62e/recipes/muparser/all/conanfile.py#L18

```suggestion
               "vtk_smp_implementation": [None, "openmp", "tbb", "stl"],
```

https://github.com/conan-io/conan-center-index/blob/41d4fbb8c218f9d938ccb8b9d888bff52002e62e/recipes/libgettext/all/conanfile.py#L18

There's no good choice for `std::thread`